src/pages/LogView/LoadingPage/useResolveLogURLAndRenderingType.ts

```diff
@@ -194,12 +199,19 @@ export const useResolveLogURL = ({
     default:
       break;
   }
+
+  // renderingType is only propagated for evergreen test logs as of DEVPROD-1435.
+  const renderingType =
+    (testData?.task?.tests.testResults[0]?.logs
```

Comment:
`renderingType` will default to `default` for resmoke logs which is confusing. I think we should try to be consistent about how this value is treated and used.